flang/runtime/command.cpp

```diff
@@ -239,4 +241,22 @@ std::int32_t RTNAME(GetEnvVariable)(const Descriptor &name,
   return StatOk;
 }
 
+std::int32_t RTNAME(GetCwd)(
+    const Descriptor &cwd, const char *sourceFile, int line) {
+  Terminator terminator{sourceFile, line};
+
+  RUNTIME_CHECK(terminator, IsValidCharDescriptor(&cwd));
+
+  char *buf = getcwd(nullptr, 0);
+  if (!buf) {
+    return StatMissingCurrentWorkDirectory;
+  }
+
+  std::int64_t strLen = StringLength(buf);
```

Comment:
Use braced initialization in the runtime.  It is safer, especially with integer types.